WordPress/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         maven { url 'https://maven.fabric.io/repo' }
```

Comment:
Can we change from `mavenCentral()` to `jcenter()`? It's the new default for Android Studio and it also seems as if they have the PhotoView version 1.2.4 we need: https://bintray.com/pixty/maven/PhotoView/view